test/conformance/tests/sriov_operator.go

```diff
@@ -939,9 +942,132 @@ var _ = Describe("[sriov] operator", func() {
+							if strings.Contains(pod.ObjectMeta.Name, resourceName) {
+								return false
+							}
+						}
+						return true
+					}, 1*time.Minute, 10*time.Second).Should(BeTrue())
+
+					Eventually(func() bool {
+						serviceList, err := clients.Services(operatorNamespace).List(metav1.ListOptions{})
+						Expect(err).ToNot(HaveOccurred())
+						for _, svc := range serviceList.Items {
+							if strings.Contains(svc.ObjectMeta.Name, resourceName) {
+								return false
+							}
+						}
+						return true
+					}, 1*time.Minute, 10*time.Second).Should(BeTrue())
+
+					Eventually(func() bool {
+						crs := rbacv1.ClusterRoleList{}
+						err = clients.List(context.Background(), &crs, runtimeclient.InNamespace("openshift-sriov-network-operator"))
+						Expect(err).ToNot(HaveOccurred())
+						for _, cr := range crs.Items {
+							if strings.Contains(cr.ObjectMeta.Name, resourceName) {
+								return false
+							}
+						}
+						return true
+					}, 1*time.Minute, 10*time.Second).Should(BeTrue())
+
+					Eventually(func() bool {
+						crbs := rbacv1.ClusterRoleBindingList{}
+						err = clients.List(context.Background(), &crbs, runtimeclient.InNamespace("openshift-sriov-network-operator"))
+						Expect(err).ToNot(HaveOccurred())
+						for _, crb := range crbs.Items {
+							if strings.Contains(crb.ObjectMeta.Name, resourceName) {
+								return false
+							}
+						}
+						return true
+					}, 1*time.Minute, 10*time.Second).Should(BeTrue())
+
+					Eventually(func() bool {
+						mwc := &admission.MutatingWebhookConfiguration{}
+						err = clients.Get(context.Background(), runtimeclient.ObjectKey{Name: resourceName + "-config", Namespace: operatorNamespace}, mwc)
+						return err != nil && strings.Contains(fmt.Sprintf("%s", err), "not found")
+					}, 1*time.Minute, 10*time.Second).Should(BeTrue())
+
+					Eventually(func() bool {
+						cms := corev1.ConfigMapList{}
+						err = clients.List(context.Background(), &cms, runtimeclient.InNamespace("openshift-sriov-network-operator"))
+						Expect(err).ToNot(HaveOccurred())
+						for _, cm := range cms.Items {
+							if strings.Contains(cm.ObjectMeta.Name, resourceName) {
+								return false
+							}
+						}
+						return true
+					}, 1*time.Minute, 10*time.Second).Should(BeTrue())
+				},
+				//25814
+				Entry("Network resource injector", "network-resources-injector", false),
+				//25847
+				Entry("Webhook resource injector", "operator-webhook", false))
+		})
 	})
 })
 
+func setSriovOperatorSpecFlag(flagName string, flagValue bool) {
+	cfg := sriovv1.SriovOperatorConfig{}
+	err := clients.Get(context.TODO(), runtimeclient.ObjectKey{
+		Name:      "default",
+		Namespace: operatorNamespace,
+	}, &cfg)
+
+	Expect(err).ToNot(HaveOccurred())
+	Expect(*cfg.Spec.EnableInjector).To(BeTrue())
+	Expect(*cfg.Spec.EnableOperatorWebhook).To(BeTrue())
+
+	if flagName == "network-resources-injector" && *cfg.Spec.EnableInjector != flagValue {
```

Comment:
We will have  "network-resources-injector" and "operator-webhook" in multiple places now, maybe use a constant?

Reply:
Good idea. Agree